pwnlib/commandline/cyclic.py

```diff
@@ -70,7 +70,7 @@ def main(args):
             pat = int(pat, 0)
         except ValueError:
             pass
-        pat = flat(pat)
+        pat = flat(pat, word_size=8 * args.length)
```

Comment:
This should set `context.cyclic_size`. instead of doing this here.

Reply:
That doesn't actually work:
```
AttributeError: cyclic pattern size cannot be larger than word size
```

but I can set `context.bytes` to `args.length` and it seems to work.

Reply:
Can we at least do `bytes=` to avoid the unnecessary multiplication?  Also this should be rebased since v4.2.0 was released and now this targets v4.4.0dev0

Reply:
Done.